CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Enh #784: Remove unused code in `AbstractSchema::getTableIndexes()` (@vjik)
 - Bug #788: Fix casting integer to string in `AbstractCommand::getRawSql()` (@Tigrov)
 - Enh #789: Remove unnecessary type casting to array in `AbstractDMLQueryBuilder::getTableUniqueColumnNames()` (@Tigrov)
+- Enh #794: Add category to log context (@darkdef)
```

Comment:
```suggestion
- Enh #794: Add message type to log context (@darkdef)
```

Or need rename in code "type" to "category".

Reply:
+